### PR TITLE
LSR-826 Relogin user

### PR DESCRIPTION
**Pull request by nhpatt**

### CODE REVIEW

**Comment by nhpatt** [comment]
👍 (tested in the sample app)

File: android/library/src/main/java/com/liferay/mobile/screens/context/SessionContext.java [hunk 1]
```diff
@@ -125,6 +133,40 @@ public static void loadStoredCredentials(CredentialsStorageBuilder.StorageType s
...
+		Executor.execute(new Runnable() {
+			@Override
+			public void run() {
+				try {
+					CurrentUserConnector currentUserConnector = ServiceProvider.getInstance().getCurrentUserConnector(session);
+					JSONObject jsonObject = currentUserConnector.getCurrentUser();
+					if (jsonObject != null) {
+
+						User user = new User(jsonObject);
+						_currentUser = user;
```
Comment (jmnavarro) [inline android/library/src/main/java/com/liferay/mobile/screens/context/SessionContext.java]: @nhpatt if the user was read from the store, that stored data is not updated with the new user retrieved.

File: android/library/src/main/java/com/liferay/mobile/screens/auth/login/connector/UserConnector.java [hunk 2]
```diff
@@ -18,5 +18,7 @@
...
 
+	JSONObject updateUser(long userId, String oldPassword, String newPassword1, String newPassword2, boolean passwordReset, String reminderQueryQuestion, String reminderQueryAnswer, String screenName, String emailAddress, long facebookId, String openId, String languageId, String timeZoneId, String greeting, String comments, String firstName, String middleName, String lastName, int prefixId, int suffixId, boolean male, int birthdayMonth, int birthdayDay, int birthdayYear, String smsSn, String aimSn, String facebookSn, String icqSn, String jabberSn, String msnSn, String mySpaceSn, String skypeSn, String twitterSn, String ymSn, String jobTitle, JSONArray groupIds, JSONArray organizationIds, JSONArray roleIds, JSONArray userGroupRoles, JSONArray userGroupIds, JSONObjectWrapper serviceContext) throws Exception;
```
Comment (jmnavarro) [inline android/library/src/main/java/com/liferay/mobile/screens/auth/login/connector/UserConnector.java]: @nhpatt I don't see where it's used

**Comment by jmnavarro** [comment]
👍

**Pull request merged**